versioned_docs/version-0.5/example/assets/install-gcp.svg

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If we already change these docs, could you please also replace `voltron-system` to `capact-system`? 🙈 
![Screenshot 2022-01-26 at 09 23 51](https://user-images.githubusercontent.com/7155799/151128019-c2e5d901-0047-4cdb-a5d3-5bf56fbee592.png)